CSS/style.css

```diff
@@ -3,23 +3,28 @@ body{
   /* background-image: :  */
   background-color: #f4f4f4;
   font-family:'Oswald', sans-serif;
-  width: 90%;
+  width: 960px;
+  position: relative;
+  text-align: 0 auto;
 }
 
 header { 
-  font-family: 'Eater', cursive; 
+  font-family: 'Eater', cursive;
+  color: #18121E;
 }
 
 section{
-  font-size: 14px
+  font-size: 14px;
 }
 
 aside {
-  width: 30%;
+  width: 300px;
+  position: align-right;
```

Comment:
Here it looks like you are wanting to attach the aside to the right of the window. The position would either need to be absolutely positioned relative to its parent, or you could float the aside right.